sparse-10/etc/dconf/db/vendor.d/lipstick-configs.txt

```diff
@@ -1,6 +1,3 @@
-[desktop/lipstick-jolla-home]
-reboot_warning_on_sim_remove=true
-
 [desktop/lipstick-jolla-home/peekfilter]
 boundaryWidth=48
 pressDelay=800
```

Comment:
This part could also be moved out to sparse-sony-10, if this PR gets merged https://github.com/mer-hybris/droid-hal-configs/pull/215